crates/hyperswitch_connectors/src/connectors/fiuu/transformers.rs

```diff
@@ -1,226 +1,837 @@
-                let card = FiuuCard {
-                    number: req_card.card_number,
-                    expiry_month: req_card.card_exp_month,
-                    expiry_year: req_card.card_exp_year,
-                    cvc: req_card.card_cvc,
-                    complete: item.router_data.request.is_auto_capture()?,
-                };
-                Ok(Self {
-                    amount: item.amount.clone(),
-                    card,
-                })
-            }
+            PaymentMethodData::Card(req_card) => Ok(Self {
+                merchant_id,
+                reference_no,
+                txn_type: match item.router_data.request.is_auto_capture()? {
+                    true => TxnType::Sals,
+                    false => TxnType::Auts,
+                },
+                txn_channel: TxnChannel::Creditz,
+                txn_currency,
+                txn_amount,
+                signature,
+                cc_pan: req_card.card_number,
+                cc_cvv2: req_card.card_cvc,
+                cc_month: req_card.card_exp_month,
+                cc_year: req_card.card_exp_year,
+                non_3ds: match item.router_data.is_three_ds() {
+                    false => 1,
+                    true => 0,
+                },
+                return_url: item.router_data.request.router_return_url.clone(),
+            }),
             _ => Err(errors::ConnectorError::NotImplemented("Payment methods".to_string()).into()),
         }
     }
 }
 
-//TODO: Fill the struct with respective fields
-// Auth Struct
-pub struct FiuuAuthType {
-    pub(super) api_key: Secret<String>,
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct FiuuPaymentsSuccessResponse {
+    #[serde(rename = "MerchantID")]
+    pub merchant_id: String,
+    pub reference_no: String,
+    #[serde(rename = "TxnID")]
+    pub txn_id: String,
+    pub txn_type: String,
+    pub txn_currency: String,
+    pub txn_amount: String,
+    pub txn_channel: String,
+    pub txn_data: TxnData,
 }
 
-impl TryFrom<&ConnectorAuthType> for FiuuAuthType {
-    type Error = error_stack::Report<errors::ConnectorError>;
-    fn try_from(auth_type: &ConnectorAuthType) -> Result<Self, Self::Error> {
-        match auth_type {
-            ConnectorAuthType::HeaderKey { api_key } => Ok(Self {
-                api_key: api_key.to_owned(),
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(untagged)]
+pub enum FiuuPaymentsResponse {
+    Success(Box<FiuuPaymentsSuccessResponse>),
+    Error(FiuuErrorResponse),
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct TxnData {
+    #[serde(rename = "RequestURL")]
+    pub request_url: String,
+    pub request_type: RequestType,
+    pub request_data: RequestData,
+    pub request_method: String,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub enum RequestType {
+    REDIRECT,
+    RESPONSE,
+}
```

Comment:
Nit: Can we use `PascalCase` names instead with `serde(rename_all = "UPPERCASE")` (or screaming snake case, as required)?